ui/packages/tabs/src/Tabs.svelte

```diff
@@ -43,26 +43,116 @@
 	}
 
 	$: selected !== null && change_tab(selected);
+	let h = 0;
```

Comment:
Can we add a scroll bar to the tabs? I found it difficult to scroll through all the tabs in the deployed demo manually with the current behavior.

Reply:
Adding a scroll bar breaks the tab design because of where it appears. I can revert the change tho until we have a better design, if that is preferable.

Reply:
I vote we revert this change until we have a better design. Happy to keep if others feel strongly about it though!

Reply:
and done!

Reply:
Thank you 🙌